lib/rouge/lexers/slice.rb

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*- #
+
```

Comment:
Can you add '# frozen_string_literal: true'?

Reply:
Done